dbt/adapters/trino/connections.py

```diff
@@ -131,8 +131,8 @@ def method(self):
 
     def trino_auth(self):
         return trino.auth.CertificateAuthentication(
```

Comment:
Let's add a unit test to verify the output of `trino_auth()` in
  `tests.unit.test_adapter.TestTrinoAdapterAuthenticationMethods.test_certificate_authentication`